config/config.go

```diff
@@ -78,7 +78,9 @@ type (
 	}
 
 	S3 struct {
-		Enabled bool `yaml:"enabled"`
+		DisableAuth bool   `yaml:"disableAuth"`
+		Enabled     bool   `yaml:"enabled"`
+		KeypairsV4  string `yaml:"keypairsV4"`
```

Comment:
Can this be a `[]string`? You'll have to add separate parsing for the environment variable, but it would make it easier to configure in yaml:
```yml
s3:
  keypairsV4:
    - foo
    - bar
```